test/jobs/auto-pause-jobs.ts

```diff
@@ -70,17 +70,19 @@ function previewingToPauseTest(username: string): void {
       const workItemQueryCmr = JSON.parse(resQueryCmr.text).workItem;
       workItemQueryCmr.status = WorkItemStatus.SUCCESSFUL;
       workItemQueryCmr.results = [
-        'test/resources/worker-response-sample/catalog0.json',
+        getStacLocation(workItemQueryCmr, 'catalog.json'),
       ];
+      await fakeServiceStacOutput(workItemQueryCmr.jobID, workItemQueryCmr.id);
       await updateWorkItem(this.backend, workItemQueryCmr);
 
       const resServExample = await getWorkForService(this.backend, 'harmonyservices/service-example:latest');
       expect(resServExample.status).to.equal(200);
       const workItemServExample = JSON.parse(resServExample.text).workItem;
       workItemServExample.status = WorkItemStatus.SUCCESSFUL;
       workItemServExample.results = [
-        'test/resources/worker-response-sample/catalog0.json',
```

Comment:
I see you deleted the sample files. I'm using them for new tests. Can I just use getStacLocation? How do I differentiate between the different files for things like:

```
      workItem.results = [
        'test/resources/worker-response-sample/catalog0.json',
        'test/resources/worker-response-sample/catalog1.json',
        'test/resources/worker-response-sample/catalog2.json',
      ];
```

EDIT: Nevermind I saw in later tests I can use:

```
      workItem.results = [
        getStacLocation(workItem, 'catalog0.json'), 
        getStacLocation(workItem, 'catalog1.json'), 
        getStacLocation(workItem, 'catalog2.json'),
      ];
```